README.md

```diff
@@ -344,6 +359,16 @@ See @tbl-pipe.
 
 See [Quarto tables](https://quarto.org/docs/authoring/tables.html)
 
+## Loading `.Rmd` files as Jupyter notebooks
+
+For this, you will need [jupytext](https://github.com/mwouts/jupytext):
```

Comment:
Doesn't this come with `build-init` and `build_requirements.txt`?

Reply:
Ah, yes; then jupyterlab is the missing component.